lib/Zureg/Views.hs

```diff
@@ -220,4 +230,4 @@ fileScanner :: B.ByteString
 fileScanner = $(Embed.embedFile "static/scanner.js")
 
 scan :: Registrant -> H.Html
-scan = registrantInfo
+scan registrant = registrantInfo registrant <> "Pick up T-Shirt later"
```

Comment:
I would use another `p` element, and put the message near the top, e.g.:

```haskell
scan registrant = do
    H.p "Pick up T-Shirt later"
    registrantInfo registrant
```

Reply:
Thank you for the comments, I have incorporated them in the next commit.